service/pagerduty/pager_duty.go

```diff
@@ -0,0 +1,70 @@
+package pagerduty
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/PagerDuty/go-pagerduty"
+	"github.com/nikoksr/notify"
+)
+
+type Client interface {
+	CreateIncidentWithContext(ctx context.Context, from string, options *pagerduty.CreateIncidentOptions) (*pagerduty.Incident, error)
+}
```

Comment:
```suggestion
type Client interface {
    CreateIncidentWithContext(ctx context.Context, from string, options *pagerduty.CreateIncidentOptions) (*pagerduty.Incident, error) //nolint:lll // acceptable in this case, alternative makes the interface even less readable
}
```

For the lll complaint, I'm fine if we simply ignore the warning. I checked the alternatives and they all make the interface borderline unreadable.